cluster-sync/sync.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+export CDI_INSTALL_TIMEOUT=${CDI_INSTALL_TIMEOUT:-120}
+
+source ./cluster-up/hack/common.sh
+source ./cluster-up/cluster/${KUBEVIRT_PROVIDER}/provider.sh
+source ./cluster-sync/${KUBEVIRT_PROVIDER}/provider.sh
+
+# Install CDI
+install_cdi
+
+wait_cdi_crd_installed $CDI_INSTALL_TIMEOUT
+
+# Install CDI CR
+echo "Waiting 480 seconds for CDI to become available"
+_kubectl wait cdis.cdi.kubevirt.io/cdi --for=condition=Available --timeout=480s
+
+install_kubevirt
+
+configure_storage
+
+# We skip the functional test additions for external provider for now, as they're specific
+if [ "${KUBEVIRT_PROVIDER}" != "external" ]; then
+  _kubectl apply -f "./cluster-sync/imageio/imageio.yaml"
+  _kubectl wait deploy/imageio-deployment -n cdi --for=condition=Available --timeout=480s
+fi
+
```

Comment:
Do we want to install `cluster-network-addons-operator ` here?

Reply:
When we start tests multus, then definitelly.